html/pfappserver/lib/pfappserver/Form/Config/Realm.pm

```diff
@@ -85,6 +86,29 @@ has_field 'admin_strip_username' =>
              help => 'Should the usernames matching this realm be stripped when used on the administration interface' },
   );
 
+has_field 'permit_userPrincipalName' =>
+  (
+   type => 'Toggle',
+   checkbox_value => "enabled",
+   unchecked_value => "disabled",
+   default => "enabled",
```

Comment:
Should default to disabled actually, since when its empty in the config file it will not be enabled. And anyway I think we're better off keeping backward compatibility where the upn can only be used for machine auth